File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("\n\nEnter pyramid height (1-8):\n\n");
    }
    while (height < 1 || height > 8);
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

# Repeatedly prompts for height until valid entry received
while True:
    height = get_int("Height: ")
    if 1 <= height <= 8:
        break

for i in range(1, height + 1):
    # Print spaces
    print(" " * (height - i), end="")

    # Print hashes
    print("#" * i)
